_rules/block-collapsible-3e12e1.md

```diff
@@ -74,20 +76,20 @@ In this document, the [visibility][visible] and [inclusion in the accessibility
 		<a href="#" onclick="toggleHidden('chapters-navigation')">Toggle table of content</a>
 
 		<nav id="chapters-navigation">
-			<h1>Content</h1>
+			<b>Content</b>
```

Comment:
I would much prefer we take out this element all together. This is a 1.3.1 issue otherwise.

```suggestion
```